libraries/testing/include/eosio/testing/tester.hpp

```diff
@@ -66,8 +66,10 @@ namespace eosio::testing {
    enum class setup_action {
       preactivate_protocol_feature,
       set_before_producer_authority_bios_contract,
-      activate_features,     // activate features listed in `setup_policy::features`. enforce dependencies.
-      activate_all_features, // activate all available features
+      activate_features,       // activate features listed in `setup_policy::features`. enforce dependencies.
+      activate_features_up_to, // activate all features up to the first one specified in `setup_policy::features`
+                               // if `setup_policy::features` contains more than one the remaining are also activated
```

Comment:
How do you determine the set of protocol features before a particular one? By dependencies?